node/src/chain_spec/oak.rs

```diff
@@ -89,6 +92,81 @@ pub fn oak_development_config() -> ChainSpec {
 	)
 }
 
+pub fn oak_staging() -> ChainSpec {
+	// Give your base currency a unit name and decimal places
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), TOKEN_SYMBOL.into());
+	properties.insert("tokenDecimals".into(), TOKEN_DECIMALS.into());
+	properties.insert("ss58Format".into(), STAGING_SS_58_FORMAT.into());
+
+	ChainSpec::from_genesis(
+		// Name
+		"Oak Staging",
+		// ID
+		"oak",
+		ChainType::Live,
+		move || {
+			let allocation_json = &include_bytes!("../../../distribution/oak_staging_alloc.json")[..];
+			let initial_allocation: Vec<(AccountId, Balance)> =
+				serde_json::from_slice(allocation_json).unwrap();
+			const ALLOC_TOKENS_TOTAL: u128 = DOLLAR * 1_000_000_000;
+			validate_allocation(
+				initial_allocation.clone(),
+				ALLOC_TOKENS_TOTAL,
+				EXISTENTIAL_DEPOSIT,
+			);
+
+			testnet_genesis(
+				// initial collators.
+				vec![
+					(
+						// SS58 prefix substrate: 5EvKT4iWQ5gDnRhkS3d254RVCfaQJSyXHj5mA4kQTsCjWchW
+						hex!["7e4f4efde71551c83714fb3062724067e7bb6ebc3bf942813321f1583e187572"]
```

Comment:
all of these invulns, sudo, council, etc are all from turing staging. do we want different accounts?

Reply:
We do; not blocking this pr. I will change those after I gathered all wallets

Reply:
cool, sounds good - just fyi, can't do genesis things for staging until this is in tho!

Reply:
sudo should be `5C571x5GLRQwfA3aRtVcxZzD7JnzNb3JbtZEvJWfQozWE54K`

Reply:
yes, i believe it should be! double check me, but i'm pretty sure that's what it's set as! (it's the same as turing staging)